code/script.js

```diff
@@ -0,0 +1,102 @@
+// DOM SELECTORS
+const projectsContainer = document.getElementById('projects')
+const header = document.getElementById('header')
+const profile = document.getElementById('profile')
+
+
+// GLOBAL VARIABLES
+const username = 'joannalodell19'
+let filteredRepo = [] // Probably not needed
+
+// let reponame = ''
+const API_URL = `https://api.github.com/users/${username}/repos`
+const API_PROFILE = `https://api.github.com/users/${username}`
+
+const API_TOKEN = TOKEN || process.env.API_KEY;
+
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: `token ${API_TOKEN}`
+    }
+}
+
+const profilePic = () => {
+    fetch(API_PROFILE)
+        .then(res => res.json())
+        .then(data => {
+            // console.log(data)
```

Comment:
To make the code more clean, try and remember to remove console.logs before sending it. I miss these all the time aswell so I'm not critizicing in any way. I belive they will be much harder on us removing these further on.